pkg/version/version_test.go

```diff
@@ -0,0 +1,100 @@
+package version
```

Comment:
ditto here, I think we can make this `version_test`

Reply:
fixed